sql.txt

```diff
@@ -1,14 +1,41 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers
+	ORDER BY Country
+	LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers
+	     WHERE ContactName LIKE 'A%';
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails
+		 WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+       VALUES (NULL, "Milky Milk", 20, 4, "500 ml", 20),
+       		  (NULL, "Candiest Candy", 8, 8, "10 kg", 42),
+              (NULL, "Tofu Taffy", 4, 7, "40 - 100 g pkgs.", 28);
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products 
+	SET Price = Price + 1.5 
+	WHERE ProductID IN(
+        SELECT ProductID
```

Comment:
First time I've seen a student submit a subquery! Very nice.

Reply:
Thank you so much! Limit was not working well with Update, so was looking for other solutions)